0.5.17 drops support for node 10 and webpack

The latest version `0.5.17` has a [breaking change](https://github.com/evanw/node-source-map-support/pull/269) for consumers who use webpack and are on node v10.x.

I've solved this by locking my version to `0.5.16`.
```
Error: ./node_modules/source-map-support/source-map-support.js
--
  | Module not found: Error: Can't resolve 'worker_threads' in '.../node_modules/source-map-support'
  | resolve 'worker_threads' in '.../node_modules/source-map-support'
  | Parsed request is a module
```

--- a/source-map-support.js
+++ b/source-map-support.js
@@ -14,6 +14,16 @@ try {
 
 var bufferFrom = require('buffer-from');
 
+/**
+ * Requires a module which is protected against bundler minification.
+ *
+ * @param {NodeModule} mod
+ * @param {string} request
+ */
+function dynamicRequire(mod, request) {
+  return mod.require(request);
+}
+
 // Only install once if called multiple times
 var errorFormatterInstalled = false;
 var uncaughtShimInstalled = false;
@@ -565,9 +575,8 @@ exports.install = function(options) {
     // Worker threads. Workers pass the error to the main thread as an event,
     // rather than printing something to stderr and exiting.
     try {
-      // Don't let browserify try to resolve this require(), it's pointless
-      // and breaks the build process.
-      var worker_threads = require('worker_' + 'threads');
+      // We need to use `dynamicRequire` because `require` on it's own will be optimized by WebPack/Browserify.
+      var worker_threads = dynamicRequire(module, 'worker_threads');
       if (worker_threads.isMainThread === false) {
         installHandler = false;
       }

